conda_smithy/configure_feedstock.py

```diff
@@ -1506,6 +1507,7 @@ def main(
 
     loglevel = os.environ.get("CONDA_SMITHY_LOGLEVEL", "INFO").upper()
     logger.setLevel(loglevel)
+    logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
Does this mean that errors are also printed to stdout instead of stderr?

Reply:
I don't know. Is all stderr output done through the logger? With this change, any log output at/above severity of `loglevel` will be printed to this handler.

It is possible to have multiple handlers and filters:

https://github.com/conda/conda-build/blob/ab17db8bdd92ddc290dba5fd3acf48133d33ba05/conda_build/utils.py#L1560-L1569